stats/memtier

```diff
@@ -99,13 +99,13 @@ while (line = $stdin.gets)
 end
 
 def gen_output_sum(type, histo_sum)
-  (1..UNITS.size - 1).each do |i|
+  (1...UNITS.size).each do |i|
     puts "total_#{type}_#{UNITS[i]}: #{histo_sum[i]}"
   end
 end
 
 def gen_output_avg(type, histo_sum, histo_num)
-  (1..UNITS.size - 1).each do |i|
+  (1...UNITS.size).each do |i|
     avg = histo_sum[i] / histo_num[i]
     puts "avg_#{type}_#{UNITS[i]}: #{avg}"
   end
```

Comment:
thanks, help merge this commit to first commit, then can do a force push of the branch, so to have 1 final commit for the PR.